src/components/AppContent/AppContent.js

```diff
@@ -20,29 +25,29 @@ const AppContent = (props) => {
     <Content>
       <div className="bx--grid">
         <div className="bx--row app-content__row">
-          <label className="app-content__label">
-            <ReadOnlyCheckbox
-              className="app-content__readonly-checkbox"
-              type="checkbox"
-              checked={readOnlyModeState}
-              onChange={readOnlyModeClickHandler}
-            />
-            ReadOnly Mode
-          </label>
+          <div>
+            <label className="app-content__label">
+              <ReadOnlyCheckbox
+                className="app-content__readonly-checkbox"
+                type="checkbox"
+                checked={readOnlyModeState}
+                onChange={readOnlyModeClickHandler}
+              />
+              ReadOnly Mode
+            </label>
+          </div>
+          {isSomeCardChecked && (
+            <DeleteButton
+              deleteCardsHandler={props.deleteCardsHandler}
+            ></DeleteButton>
+          )}
         </div>
```

Comment:
I would suggest to mmove this part to a separate component - Actionbar or Cockpit or whatever.